src/Symfony/Component/Translation/Bridge/Loco/LocoProvider.php

```diff
@@ -33,14 +34,16 @@ final class LocoProvider implements ProviderInterface
     private HttpClientInterface $client;
     private LoaderInterface $loader;
     private LoggerInterface $logger;
+    private TranslatorBagInterface $translatorBag;
```

Comment:
Hum, I only need to get a `MessageCatalogueInterface` and method `TranslatorBagInterface::getCatalogue()` is enough.

Also I prefer depending of an interface instead of an implementation, WDYT?